src/Fields/mapreduce.jl

```diff
@@ -67,24 +67,32 @@ end
 """
     norm(v::Field, p=2; normalize=true)
 
-The approximate ``L^p`` norm of `v`. This is computed by summation of the norms
-of the field values multiplied by the Jacobians and the quadrature weights. If
-`normalize=true` (the default), then it is divided by the sum of the weights:
+The approximate ``L^p`` norm of `v`, where ``L^p`` represents the space of measurable
+functions for which the p-th power of the absolute value is Lebesgue integrable, that is:
 ```math
-\\left(\\frac{\\sum_i \\|v_i\\|^p W_i J_i}{\\sum_i W_i J_i}\\right)^{1/p}
+\\| v \\|_p = \\left( \\int_\\Omega |v_i|^p d \\Omega \\right)^{1/p}
+```
+where ``|v_i|`` is defined to be the absolute value if ``v` is a scalar-valued Field, or the 2-norm
```

Comment:
```suggestion
\\| v \\|_p = \\left( \\int_\\Omega |v|^p d \\Omega \\right)^{1/p}
```
where ``|v|`` is defined to be the absolute value if ``v` is a scalar-valued Field, or the 2-norm
```